### ISSUE TITLE
Does not start up with HA 2023.3.0

### ISSUE THREAD

**Issue by BebeMischa** [dolezsa/thermal_comfort#280]
Hello,

after today's update of HA Thermal Comfort won't start up anymore.

Get this in the log:

```Logger: homeassistant.components.sensor
Source: custom_components/thermal_comfort/sensor.py:423
Integration: Sensor (documentation, issues)
First occurred: 21:55:48 (1 occurrences)
Last logged: 21:55:48

Error while setting up thermal_comfort platform for sensor
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/helpers/entity_platform.py", line 293, in _async_setup_platform
    await asyncio.shield(task)
  File "/config/custom_components/thermal_comfort/sensor.py", line 420, in async_setup_platform
    sensors += [
  File "/config/custom_components/thermal_comfort/sensor.py", line 423, in <listcomp>
    entity_description=SensorEntityDescription(
TypeError: SensorEntityDescription.__init__() got an unexpected keyword argument 'native_precision'
```

Can you please have a look on it?

Thank you ;-)

**Comment by Momentum6890** [comment]
Same issue here.

**Comment by pSpitzner** [comment]
related to the new frontend-configurable sensor precision:
https://developers.home-assistant.io/blog/2023/02/08/sensor_presentation_rounding/

as a temporary workaround, you can search and replace 
`"native_precision"` with `"suggested_display_precision"` in `/config/custom_components/thermal_comfort/sensor.py`

**Comment by ProTofik** [comment]
> related to the new frontend-configurable sensor precision: https://developers.home-assistant.io/blog/2023/02/08/sensor_presentation_rounding/
> 
> as a temporary workaround, you can search and replace `"native_precision"` with `"suggested_display_precision"` in `/config/custom_components/thermal_comfort/sensor.py`

Thanks. I got the same issue, I will just wait for an update fixing it.

**Comment by Tt0x** [comment]
> as a temporary workaround, you can search and replace `"native_precision"` with `"suggested_display_precision"` in `/config/custom_components/thermal_comfort/sensor.py`

Thanks a lot @pSpitzner the fix worked perfectly fine for me!

**Comment by simonepittis** [comment]
Thanks, @pSpitzner the workaround works! waiting the final fix

**Comment by T81** [comment]
Thanks for quick resolution

**Comment by marcpepper** [comment]
Worked for me as well. Thanks so much for the quick fix!!

**Comment by pmfernandes** [comment]
Thanks @pSpitzner. Workarround worked well here.

**Comment by pmfernandes** [comment]
@dolezsa this commit https://github.com/dolezsa/thermal_comfort/commit/b8e9cc695f8b358e0e80d2a7b5ff05bf68e91534 fixes the issue.
Can you please create a new version of the addon?

**Comment by rautesamtr** [comment]
Sorry for the wait. Just did the release. I had to wait for ha 2023.3 to be released since this is a breaking change beween 2023.2 and 2023.3

**Comment by BebeMischa** [comment]
Fixed. Thanks for the super fast response ;-)
